### DISCUSSION TITLE
Website check question

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/clomonitor#1628] [Q&A]
Hi @nate-double-u 👋

The website is taken from the website set in the repository, not at the organization level.

<img width="622" alt="capsule" src="https://github.com/user-attachments/assets/133924e0-3b03-43b0-bd61-3ac47f711a6e">

_Originally posted by @tegioz in https://github.com/cncf/clomonitor/issues/1615#issuecomment-2446044233_
            

Which repo, or checks set are responsible for pulling in the website info?

[Volcano](https://clomonitor.io/projects/cncf/volcano) is failing the website check, but it's main code repo has a URL set: https://github.com/volcano-sh/volcano

**Comment by tegioz** [comment]
Hi @nate-double-u

The `documentation/website` check is part of the [community check set](https://clomonitor.io/docs/topics/checks/).

**Reply by nate-double-u** [reply]
I think this is still an issue, not a discussion (though, admittedly the issue may have been mis-titled). [Volcano is still failing it's website check](https://clomonitor.io/projects/cncf/volcano), despite the maintainers having set the URL in GitHub in at least two places (https://github.com/volcano-sh/volcano, https://github.com/volcano-sh/website). We should expand the check sets to include the org level if set, and often the docs check will be looking at a website, so should also include the website check.
